templates/cookiecutter/README.md

```diff
@@ -0,0 +1,91 @@
+# Using the `cookiecutter` to generate models
```

Comment:
The "the" is weird. Either "Using `cookiecutter` to generate models" or "Using the `cookiecutter` templates to generate models"